src/app/public/modules/country-field/country-field.component.ts

```diff
@@ -168,12 +206,28 @@ export class SkyCountryFieldComponent implements ControlValueAccessor, OnDestroy
 
   private _selectedCountry: SkyCountryFieldCountry;
 
+  private _supportedCountries: string[];
+
   constructor(
     private changeDetector: ChangeDetectorRef,
     private elRef: ElementRef,
     private windowRef: SkyAppWindowRef,
-    @Self() @Optional() private ngControl: NgControl
+    private injector: Injector
   ) {
+    this.setupCountries();
+
+    this.countrySearchFormControl = new FormControl();
+  }
+
+  /**
+   * Angular lifecycle hook for when the component is initialized
+   * @internal
+   */
+  public ngOnInit(): void {
+
+    // tslint:disable-next-line: no-null-keyword
+    this.ngControl = this.injector.get<NgControl>(NgControl as Type<NgControl>, null);
```

Comment:
@Blackbaud-AlexKingman This feels fishy to me; didn't you find out an "approved" way to do this? Don't we get the  `NgControl` in the validator callback?

Reply:
Trevor talked this over with me. Apparently, theres no way with our approved method to do validation as well as our own internal custom validation without grabbing the control through the injector.